src/main/java/uk/gov/companieshouse/api/accounts/service/impl/StatementsServiceProperties.java

```diff
@@ -13,7 +14,7 @@
     private Map<String, String> statements;
 
     public Map<String, String> getStatements() {
-        return statements;
+        return new HashMap<>(statements);
```

Comment:
Hi, is this different to initialising  **statements** like this:
**private Map<String, String> statements = new HashMap<>();**
and then just returning **statements** from the **getStatements()** method ?

Reply:
It creates a new clone of the Map every time `getStatements` is called preventing the original Map from being altered. I could rename the method `getCopyOfStatements` to make it clearer what the intent is?

Reply:
I think your suggestion for the method name is good, would getCloneOfStatements ? be more accurate?

Reply:
Agreed I will change it now.